include/dsAudio.h

```diff
@@ -2063,6 +2063,30 @@ dsError_t  dsGetSecondaryLanguage(intptr_t handle, char* sLang);
  */
 dsError_t dsGetHDMIARCPortId(int *portId);
 
+/**
+* @brief Sets the Mixer Volume level for the given input
```

Comment:
Describe clearly that this functionality is for HDMI based input alone and also provide bit more information on input stream (MS12).

Reply:
This is not used HDMI Input . It is basically used for controlling volume levels or mixer levels for each of the Primary and System Input

Reply:
Can you define what do mean by Primary and System Input

Reply:
updated